### PR TITLE
Add quarterly report for re@

### PR DESCRIPTION
**Pull request by cperciva**

### CODE REVIEW

**Review by pauamma — APPROVED** [review]

File: 2022q4/releng.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Contact: FreeBSD Release Engineering Team, <re@FreeBSD.org>
+
+The FreeBSD Release Engineering Team is responsible for setting and publishing release schedules for official project releases of FreeBSD, announcing code freezes and maintaining the respective branches, among other things.
+
+During the fourth quarter of 2022, the Release Engineering Team completed work on the 12.4-RELEASE cycle.
+This is the final release from the stable/12 branch.
+During the release cycle, only one BETA builds and two RC (release candidate) builds were needed; overall the release cycle went very smoothly and the release took place on December 5th.
+
+During the fourth quarter of 2022, the Release Engineering Team continued providing weekly development snapshot builds for the *main*, *stable/13*, and *stable/12* branches.
```
Comment (grahamperrin) [inline 2022q4/releng.adoc:18]: > During the fourth quarter of 2022, the Release Engineering Team

The previous paragraph began with the same phrase, and it's all in the context of a quarterly report for the specified period. 

Instead: 

> The Team

File: 2022q4/releng.adoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+link:https://www.freebsd.org/releases/12.4R/announce/[FreeBSD 12.4-RELEASE announcement] URL: link:https://www.freebsd.org/releases/12.4R/announce/[https://www.freebsd.org/releases/12.4R/announce/] +
+link:https://www.freebsd.org/releases/13.2R/schedule/[FreeBSD 13.2-RELEASE schedule] URL: link:https://www.freebsd.org/releases/13.2R/schedule/[https://www.freebsd.org/releases/13.2R/schedule/] +
+link:https://www.freebsd.org/releases/14.0R/schedule/[FreeBSD 14.0-RELEASE schedule] URL: link:https://www.freebsd.org/releases/14.0R/schedule/[https://www.freebsd.org/releases/14.0R/schedule/] +
+link:https://download.freebsd.org/snapshots/ISO-IMAGES/[FreeBSD development snapshots] URL: link:https://download.freebsd.org/snapshots/ISO-IMAGES/[https://download.freebsd.org/snapshots/ISO-IMAGES/]
+
+Contact: FreeBSD Release Engineering Team, <re@FreeBSD.org>
+
+The FreeBSD Release Engineering Team is responsible for setting and publishing release schedules for official project releases of FreeBSD, announcing code freezes and maintaining the respective branches, among other things.
```
Comment (grahamperrin) [inline 2022q4/releng.adoc:12]: Instead: 

> Team responsibilities include setting and publishing release schedules, announcing code freezes and maintaining branches.

File: 2022q4/releng.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+During the fourth quarter of 2022, the Release Engineering Team completed work on the 12.4-RELEASE cycle.
```
Comment (grahamperrin) [inline 2022q4/releng.adoc:14]: > the Release Engineering Team completed

Instead: 

> the Team completed

File: 2022q4/releng.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+This is the final release from the stable/12 branch.
+During the release cycle, only one BETA builds and two RC (release candidate) builds were needed; overall the release cycle went very smoothly and the release took place on December 5th.
+
+During the fourth quarter of 2022, the Release Engineering Team continued providing weekly development snapshot builds for the *main*, *stable/13*, and *stable/12* branches.
+
+In the first quarter of 2023, the Release Engineering Team will start work on the upcoming 13.2-RELEASE.
```
Comment (grahamperrin) [inline 2022q4/releng.adoc:20]: > the Release Engineering Team will

Instead: 

> the Team will

Comment (grahamperrin) [reply 2022q4/releng.adoc:20]: > work on the upcoming 13.2-RELEASE.

Instead: 

> work on 13.2-RELEASE.

**Review by grahamperrin — CHANGES_REQUESTED** [review]
Singular, not plural, for one BETA. 

Whilst here: conciseness.

File: 2022q4/releng.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+link:https://www.freebsd.org/releases/14.0R/schedule/[FreeBSD 14.0-RELEASE schedule] URL: link:https://www.freebsd.org/releases/14.0R/schedule/[https://www.freebsd.org/releases/14.0R/schedule/] +
+link:https://download.freebsd.org/snapshots/ISO-IMAGES/[FreeBSD development snapshots] URL: link:https://download.freebsd.org/snapshots/ISO-IMAGES/[https://download.freebsd.org/snapshots/ISO-IMAGES/]
+
+Contact: FreeBSD Release Engineering Team, <re@FreeBSD.org>
+
+The FreeBSD Release Engineering Team is responsible for setting and publishing release schedules for official project releases of FreeBSD, announcing code freezes and maintaining the respective branches, among other things.
+
+During the fourth quarter of 2022, the Release Engineering Team completed work on the 12.4-RELEASE cycle.
+This is the final release from the stable/12 branch.
+During the release cycle, only one BETA builds and two RC (release candidate) builds were needed; overall the release cycle went very smoothly and the release took place on December 5th.
```
[the remark this answers is not in the document]
Comment (cperciva) [reply 2022q4/releng.adoc]: Fixed.

**Comment by cperciva** [comment]
Thanks for the suggestions re conciseness; but I was reusing text which appeared in previous releng reports and I'd prefer not to change the template at this time.

**Comment by grahamperrin** [comment]
@cperciva thanks, understood.

**Review by pauamma — APPROVED** [review]

**Pull request merged**